addons/invtracker/invtracker.lua

```diff
@@ -0,0 +1,460 @@
+    if (os.time() - update_time > 30 and id == 0x50 or id == 0xE0 or id == 0x1D) then
+      update()
+    end
+end)
+
+windower.register_event('add item', function(...)
+    if (os.time() - update_time > 30) then
+        update()
+    end
+end)
+
+windower.register_event('remove item', function(...)
+    if (os.time() - update_time > 30) then
+        update()
+    end
+end)
+
+function initialize()
+    hideKey = settings.HideKey
+    update()
+    show()
+end
+
+function update()
+    current_block = 0
+    last_column = 1
+    items = windower.ffxi.get_items()
+    update_equipment()
+    update_items()
+end
+
+function update_equipment()
+    if (settings.SlotImage.equipment.visible) then
+        initialize_block()
+        print_equipment(items.equipment.back)
+        print_equipment(items.equipment.waist)
+        print_equipment(items.equipment.legs)
+        print_equipment(items.equipment.feet)
+        print_equipment(items.equipment.body)
+        print_equipment(items.equipment.hands)
+        print_equipment(items.equipment.left_ring)
+        print_equipment(items.equipment.right_ring)
+        print_equipment(items.equipment.head)
+        print_equipment(items.equipment.neck)
+        print_equipment(items.equipment.left_ear)
+        print_equipment(items.equipment.right_ear)
+        print_equipment(items.equipment.main)
+        print_equipment(items.equipment.sub)
+        print_equipment(items.equipment.range)
+        print_equipment(items.equipment.ammo)
+    end
+end
+
+function update_items()
+    update_bag(settings.SlotImage.inventory,items.inventory,items.max_inventory,items.enabled_inventory)
+    update_bag(settings.SlotImage.mogSafe,items.safe,items.max_safe,items.enabled_safe)
+    update_bag(settings.SlotImage.mogSafe,items.safe2,items.max_safe2,items.enabled_safe2)
+    update_bag(settings.SlotImage.mogStorage,items.storage,items.max_storage,items.enabled_storage)
+    update_bag(settings.SlotImage.mogLocker,items.locker,items.max_locker,items.enabled_locker)
+    update_bag(settings.SlotImage.mogSatchel,items.satchel,items.max_satchel,items.enabled_satchel)
+    update_bag(settings.SlotImage.mogSack,items.sack,items.max_sack,items.enabled_sack)
+    update_bag(settings.SlotImage.mogCase,items.case,items.max_case,items.enabled_case)
+    update_bag(settings.SlotImage.mogWardrobe,items.wardrobe,items.max_wardrobe,items.enabled_wardrobe)
+    update_bag(settings.SlotImage.mogWardrobe,items.wardrobe2,items.max_wardrobe2,items.enabled_wardrobe2)
+    update_bag(settings.SlotImage.mogWardrobe,items.wardrobe3,items.max_wardrobe3,items.enabled_wardrobe3)
+    update_bag(settings.SlotImage.mogWardrobe,items.wardrobe4,items.max_wardrobe4,items.enabled_wardrobe4)
+    update_temp_bag(settings.SlotImage.tempInventory,items.treasure,#items.treasure,items.enabled_treasure)
+    --update_temp_bag(settings.SlotImage.tempInventory,items.temporary,items.max_temporary,items.enabled_temporary)
```

Comment:
Forgot to rempove that? Or forgot to uncomment it? Or WIP?

Reply:
My bad, this got commented out while testing and I've added it back with a small refactor.